rootex/core/renderer/rendering_device.cpp

```diff
@@ -333,6 +345,21 @@ void RenderingDevice::bind(ID3D11InputLayout* inputLayout)
 	m_Context->IASetInputLayout(inputLayout);
 }
 
+//Assuming subresource offset = 0
+void RenderingDevice::getBufferMappedContext(ID3D11Buffer* buffer, D3D11_MAPPED_SUBRESOURCE& subresource)
```

Comment:
Remove the word `get` from the name of this function. This doesn't get anything. It rather sets a buffer for mapping